docs/lib/datastore/fragments/js/examples.md

```diff
@@ -75,10 +76,126 @@ function App() {
 
 export default App;
 ```
+</amplify-block>
+<amplify-block name="Angular">
 
-## Example React Native application  
```

Comment:
Since the below is an Angular example, can you look to add a title from the above here?